### PR TITLE
Added first file

### PR DESCRIPTION
**Pull request by Tiberiw**
Am adauga...

### CODE REVIEW

**Comment by Domynis** [comment]
comment

**Comment by Bacter16** [comment]
bad shit

**Comment by florinacho337** [comment]
salut

**Review by Bacter16 — APPROVED** [review]

File: file_test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Test primul testtt
```
Comment (florinacho337) [inline file_test.txt:1]: vezi ca e typo colea

**Review by florinacho337 — APPROVED** [review]
perfect

**Pull request merged**